src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -447,6 +447,12 @@ public bool DisableCustomPackageLocations
         /// </summary>
         public bool ShowRunPreview { get; set; }
 
+
+        /// <summary>
+        /// Stores the group styles added in the preference settings
+        /// </summary>
+        public List<StyleItem> StyleItemsList { get; set; }
```

Comment:
Make this internal, so that we can change it with breaking in future.

Reply:
It wasn't possible to be changed to internal because it seems to not save the fields in the XML file